vector/src/main/java/im/vector/app/features/voice/VoiceRecorderL.kt

```diff
@@ -112,6 +99,14 @@ class VoiceRecorderL(
         }
     }
 
+    override fun pauseRecord() {
+        Toast.makeText(context, "Not implemented for this Android version", Toast.LENGTH_SHORT).show()
+    }
+
+    override fun resumeRecord() {
+        Toast.makeText(context, "Not implemented for this Android version", Toast.LENGTH_SHORT).show()
+    }
```

Comment:
Maybe we could add a `fun canPauseAndResume(): Boolean` in the interface to adapt the UI pro-actively. Or a more extensive `fun getRecorderCapability(): RecorderCapability`.

Reply:
I will try to consider this point in another PR